contrib/gitian-builder/gitian-build.sh

```diff
@@ -0,0 +1,414 @@
+#!/bin/bash
+
+# Copyright (c) 2016 The Bitcoin Core developers
+# Copyright (c) 2021 The Dogecoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+# GITIAN PROPERTIES
+unset USE_LXC
```

Comment:
regarding the USE_*, unfortunately yes it s mandatory because gitian-builder is not doing exclusivity for the USE. It means that if you have both USE_LXC AND USE_DOCKER, USE_LXC will prevail :(

https://github.com/devrandom/gitian-builder/search?q=USE_LXC

Reply:
But can't we assume users won't have those environment variables set ?
I guess it may be set if someone is doing another gitian build for any crypto at the same time (or keep it as env...), but may be really rare and perhaps non-existent.

Reply:
I added a condition to check if those environment variables exists :)